lib/cdo/aws/rds.rb

```diff
@@ -78,6 +78,7 @@ def self.clone_cluster(
     end
 
     def self.delete_cluster(cluster_id, max_attempts = 20, delay = 60)
+      raise StandardError.new("CLUSTER_ID environment variable is required.") unless cluster_id.present?
```

Comment:
This feels like the wrong place for this particular error message to go; this method itself doesn't know anything about an environment variable, just the parameter. I'd vote we either change the message to something like,

```suggestion
      raise StandardError.new("cluster_id is required in order to delete cluster.") unless cluster_id.present?
```

Or move this error message to wherever is actually trying to read the value from the environment.

Reply:
Agreed. Fixed!